### DISCUSSION TITLE
Adding Emoji's to Doc Titles

### DISCUSSION THREAD

**Discussion by edwarddarrah** [outline/outline#5323] [General]
Hello, 
I recently tried to add an emoji to a Doc title. However, it appears the formatting is thrown off if a space is added b/w the emoji and the title. I was wondering if this was a bug. 

Here is a short video: [https://share.cleanshot.com/M39j7s9j](https://share.cleanshot.com/M39j7s9j)

**Comment by tommoor** [comment]
This is by design, to keep the text aligned

**Reply by edwarddarrah** [reply]
Thanks for reply. It looks like it is cutting off button of emojis, depending on its size and shape.
